pkg/gofr/datasource/pubsub/mqtt/mqtt.go

```diff
@@ -19,6 +19,7 @@ import (
 const (
 	publicBroker  = "broker.hivemq.com"
 	messageBuffer = 10
+	CloseTimeout  = 250 * time.Millisecond
```

Comment:
This would be clearer, maybe and I don't think we need an exported constant

```suggestion
	defaultCloseTimeout  = 250 * time.Millisecond
```

Reply:
We are using this constant inside container's **createMqttPubSub** method, so it needs to be exported.
Will change name to `DefaultCloseTimeout`